build/deploy/schema-manager.libsonnet

```diff
@@ -28,6 +28,7 @@ local schema_dir = '/db-schemas';
               },
               volumeMounts: volumes.mounts.caCert + volumes.mounts.clientCert,
             },
+            imagePullSecrets: if metadata.image_pull_secret != null then [{name: metadata.image_pull_secret}] else [],
```

Comment:
Same here, I think that if you put this line in `base.Job` it would apply to all jobs in the project

Reply:
I will update the PodSpec which should apply to Job, Deployment and Statefulset resources.